web/src/tippyjs.js

```diff
@@ -197,6 +197,22 @@ export function initialize() {
         appendTo: () => document.body,
     });
 
+    delegate("body", {
+        target: [
+            // Ideally this would be `#compose_buttons .button`, but the
+            // reply button's actual area is its containing span.
+            "#compose_buttons > .reply_button_container",
+            "#left_bar_compose_mobile_button_big",
+            "#left_bar_compose_stream_button_big",
+            "#left_bar_compose_private_button_big",
+        ],
+        delay: EXTRA_LONG_HOVER_DELAY,
```

Comment:
Changed this to the new standard tooltip delay for buttons where the text basically matches the tooltip's.